### PR TITLE
change font from campaign to open sans

### PR DESCRIPTION
**Pull request by jdamiba**
closes https://github.com/plotly/documentation/issues/1684

### CODE REVIEW

File: all_static/css/main.css [hunk 1]
```diff
@@ -9438,69 +9438,9 @@ pre, code, .hljs {
...
-font-display:auto;font-style:normal;font-weight:700;
-}
-
-@font-face {
-font-family:"campaign";
-src:url("https://use.typekit.net/af/3fc953/00000000000000003b9afbc7/27/l?primer=7cdcb44be4a7db8877ffa5c0007b8dd865b3bbc383831fe2ea177f62257a9191&fvd=i7&v=3") format("woff2"),url("https://use.typekit.net/af/3fc953/00000000000000003b9afbc7/27/d?primer=7cdcb44be4a7db8877ffa5c0007b8dd865b3bbc383831fe2ea177f62257a9191&fvd=i7&v=3") format("woff"),url("https://use.typekit.net/af/3fc953/00000000000000003b9afbc7/27/a?primer=7cdcb44be4a7db8877ffa5c0007b8dd865b3bbc383831fe2ea177f62257a9191&fvd=i7&v=3") format("opentype");
-font-display:auto;font-style:italic;font-weight:700;
-}
+@import url("https://use.typekit.net/xib5kch.css");
```
Comment (nicolaskruchten) [inline all_static/css/main.css:9443]: do we need this line and the one above? what does each one do? cc @VeraZab ?

**Comment by nicolaskruchten** [comment]
Did you check out the Percy diffs here? no more Jubilat...

File: all_static/css/main.css (hunk 1 above)
Comment (VeraZab) [reply all_static/css/main.css:9443]: sorry, not sure what you mean by the one above?
the @import statement seems right.. i created a new adobe font project, with both jubilat and open sans in that project, and that's the right url yes.

File: all_static/css/main.css [hunk 2]
```diff
@@ -9438,69 +9438,9 @@ pre, code, .hljs {
...
- *   - http://typekit.com/eulas/00000000000000003b9afbc7
- * jubilat:
- *   - http://typekit.com/eulas/00000000000000003b9b0d24
- *   - http://typekit.com/eulas/00000000000000003b9b0d22
- *
- * © 2009-2020 Adobe Systems Incorporated. All Rights Reserved.
- */
-/*{"last_published":"2020-02-05 02:30:33 UTC"}*/
-
 @import url("https://p.typekit.net/p.css?s=1&k=dsx1rks&ht=tk&f=9882.9892.34284.34285.34286.34288.34289&a=37708615&app=typekit&e=css");
```
Comment (VeraZab) [inline all_static/css/main.css:9441]: oh you probably mean this file..well i don't know what this one is for.. @jdamiba ?

File: all_static/css/main.css [hunk 3]
```diff
@@ -8233,7 +8233,7 @@ div#notebook {
       .page.\--home .\--documentation-categories ul li h2 {
         width: 100%;
         font-size: 1.4rem;
-        font-family: "campaign", sans-serif;
+        font-family: "Open Sans", sans-serif;
```
Comment (VeraZab) [inline all_static/css/main.css:8236]: in the typekit file, Open Sans is referred to this way: "open-sans"
that's the way you have to use it in your css files

Comment (VeraZab) [reply all_static/css/main.css:8236]: also, please review, it's "open-sans", sans-serif, and "jubilat", serif, as per bottom of typekit file:
```
.tk-jubilat { font-family: "jubilat",serif; }
.tk-open-sans { font-family: "open-sans",sans-serif; }
```

File: all_static/css/main.css (hunk 1 above)
Comment (jdamiba) [reply all_static/css/main.css:9441]: I deleted the other `import` as the css it pointed to just contained a comment.

File: all_static/css/main.css (hunk 3 above)
Comment (jdamiba) [reply all_static/css/main.css]: I standardized the font declarations throughout the file.

**Comment by nicolaskruchten** [comment]
The Percy diffs look good to me now, just want to address the one comment above and we can merge.

File: all_static/css/main.css [hunk 4]
```diff
@@ -9438,69 +9438,19 @@ pre, code, .hljs {
...
-font-display:auto;font-style:normal;font-weight:700;
-}
-
-@font-face {
-font-family:"campaign";
-src:url("https://use.typekit.net/af/3fc953/00000000000000003b9afbc7/27/l?primer=7cdcb44be4a7db8877ffa5c0007b8dd865b3bbc383831fe2ea177f62257a9191&fvd=i7&v=3") format("woff2"),url("https://use.typekit.net/af/3fc953/00000000000000003b9afbc7/27/d?primer=7cdcb44be4a7db8877ffa5c0007b8dd865b3bbc383831fe2ea177f62257a9191&fvd=i7&v=3") format("woff"),url("https://use.typekit.net/af/3fc953/00000000000000003b9afbc7/27/a?primer=7cdcb44be4a7db8877ffa5c0007b8dd865b3bbc383831fe2ea177f62257a9191&fvd=i7&v=3") format("opentype");
-font-display:auto;font-style:italic;font-weight:700;
-}
+  font-family:"jubilat";
```
[the remark this answers is not in the document]
Comment (jdamiba) [reply all_static/css/main.css:9444]: I believe that yes, these lines need to be included. When they are not, as was the case in the last commit, the `jubilat` font is not rendered in the percy snapshots.

[the remark this answers is not in the document]
Comment (nicolaskruchten) [reply all_static/css/main.css:9444]: OK, can you dig into why that is please? In principle the idea behind loading the TypeKit file is that it maintains all of this for us... @VeraZab  have you seen this behaviour also?

[the remark this answers is not in the document]
Comment (VeraZab) [reply all_static/css/main.css:9444]: hmm, indeed, you should only need to use the @import statement. In Dash docs, they just copy-paste the typekit file into a typography.css file, and on the marketing website, we included the font inside of our html head, so I haven't used the @import statement on my end, but I would expect it to work and import jubilat.. Jubilat is indeed part of the fonts included when you open up that typekit link..

However, I see a difference in the font weights included in your css file Joe, and the ones imported in the typekit file. The typekit file has a 400 and 700 font weight only, but here you have 400 and 500. So is it really that the font is not getting imported without those statements, or just the fact its a different font weight it makes it look different?

[the remark this answers is not in the document]
Comment (VeraZab) [reply all_static/css/main.css:9444]: percy snapshots is one thing, but locally in the docs, were you seeing the jubilat font on titles applied or not.. when looking at the 'computed' section, 'rendered fonts'
Screen Shot 2020-04-30 at 10 23 42 AM

**Comment by VeraZab** [comment]
oh my mistake guys, in the typekit project, i didn't include the 500 font-weight, that previously used to be there.. its corrected now, you can see them in the typekit file.

so you shouldn't need those jubilat declarations anymore @jdamiba, just that import statement, could you try again?

**Comment by nicolaskruchten** [comment]
OK so now no more Jubilat in Percy. We should be able to get Jubilat in Percy just by using the style info from the typekit CSS.

**Comment by jdamiba** [comment]
Per our conversation this morning, I have reverted to inlining the same typography CSS as is used on `dash.plotly.com` here. Also made sure that the headings are in Jubliat, as discussed.

**Comment by nicolaskruchten** [comment]
Looks good, thanks!

**Pull request merged**